tests/test_probe.py

```diff
@@ -110,40 +111,48 @@ def test_probe():
     df = probe.to_dataframe(complete=False)
     other2 = Probe.from_dataframe(df)
     df = probe_3d.to_dataframe(complete=True)
-    # print(df.index)
+    # print(df.index)
     other_3d = Probe.from_dataframe(df)
     assert other_3d.ndim == 3
 
     # slice handling
-    selection = np.arange(0,18,2)
+    selection = np.arange(0, 18, 2)
     # print(selection.dtype.kind)
     sliced_probe = probe.get_slice(selection)
     assert sliced_probe.get_contact_count() == 9
-    assert sliced_probe.contact_annotations['impedance'].shape == (9, )
+    assert sliced_probe.contact_annotations["impedance"].shape == (9,)
 
-    #~ from probeinterface.plotting import plot_probe_group, plot_probe
-    #~ import matplotlib.pyplot as plt
-    #~ plot_probe(probe)
-    #~ plot_probe(sliced_probe)
+    # ~ from probeinterface.plotting import plot_probe_group, plot_probe
+    # ~ import matplotlib.pyplot as plt
+    # ~ plot_probe(probe)
+    # ~ plot_probe(sliced_probe)
 
-    selection = np.ones(24, dtype='bool')
+    selection = np.ones(24, dtype="bool")
     selection[::2] = False
     sliced_probe = probe.get_slice(selection)
     assert sliced_probe.get_contact_count() == 12
-    assert sliced_probe.contact_annotations['impedance'].shape == (12, )
+    assert sliced_probe.contact_annotations["impedance"].shape == (12,)
 
-    #~ plot_probe(probe)
-    #~ plot_probe(sliced_probe)
-    #~ plt.show()
+    # ~ plot_probe(probe)
+    # ~ plot_probe(sliced_probe)
+    # ~ plt.show()
 
 
-def test_set_shanks():
-    probe = Probe(ndim=2, si_units='um')
-    probe.set_contacts(
-            positions= np.arange(20).reshape(10, 2),
-            shapes='circle',
-            shape_params={'radius' : 5})
+def test_probe_equality_dunder():
+    probe1 = generate_dummy_probe()
+    probe2 = generate_dummy_probe()
+
+    assert probe1 == probe1
+    assert probe2 == probe2
+    assert probe1 == probe2
 
+    # Modify probe2
+    probe2.move([1, 1])
+
```

Comment:
are you missing an `assert probe2 != probe1`?

Reply:
Yes.